packages/gnome-shell-extension-installer/gnome-shell-extension-installer.pacscript

```diff
@@ -0,0 +1,21 @@
+name="Gnome Shell Extension Installer"
+pkgname="gnome-shell-extension-installer"
+version="1.7"
+description="A bash script to install and search extensions from extensions.gnome.org."
+maintainer="D-Brox <BroxZuka@protonmail.com>"
+url="https://github.com/brunelli/gnome-shell-extension-installer.git"
+build_depends="perl dbus"
+breaks="gnome-shell-extension-installer"
```

Comment:
```suggestion

```
Not required as `gnome-shell-extension-installer` isn't in the official ubuntu repos, and not installable by `apt`